00001-enarx-sprint-framework/README.md

```diff
@@ -0,0 +1,542 @@
+purpose of this step is not to express that all work is important; that's
+counterproductive and doesn't accomplish anything.
+
+A good rule of thumb is to accept half of the tasks in Proposed -- since team
+members should be promoting twice the work they can do, that should balance out
+to a roughly reasonable amount of work. Additionally, stakeholders should keep
+in mind the possibility of "drive-by" contributions and the developer time
+required to ensure they receive needed attention when thinking about task load.
+
+At this point, the Accepted column should contain a prioritized, well-sized list
+of tasks for the next sprint. The final step is to look for volunteers for each
+task. This _should_ be straightforward, given team member specialties and focus
+areas. If, for whatever reason, a team member doesn't think they can handle the
+workload of an individual task, this allows them one final opportunity to opt
+out (in good faith). The expectation should be that the vast majority of tasks
+get volunteers.
+
+Once a task has a volunteer, they should get assigned to it. From there,
+automation will handle moving it into the correct board column.
+
+Once this has been repeated for all tasks in Accepted, the workload for the
+coming sprint has been defined, and sprint planning is complete.
+
+This meeting should be scheduled for the hour immediately following the
+retrospective. The two combined meetings should collectively replace the daily
+standup on the first day of a new sprint. Here's a possible time breakdown:
+
+- 10 min: triage
+- 5 min: last-minute issue nomination
+- 30 min: discussion of all nominated issues and whether they are project
+  priorities 
+- 15 min: volunteering and assignment
+
+### Daily Standup
+
+Daily standup is the team's daily check-in meeting, and when the **Triage**
+process is performed. It should provide an opportunity for individual
+contributors (and guests) to ask questions, elaborate on blockers, and ask for
+help if needed.
+
+As noted before, triage is the single most important component of this meeting,
+and should therefore be done first. The exact process of triage is detailed
+separately in the [Triage](#triage) section. Triage should not be strictly
+time-bound; if productive discussions are happening around a new issue or PR,
+those should be allowed to happen naturally.
+
+The decision to make triage a part of daily standup is deliberate: it allows
+issues and PRs filed by "drive-by" contributors (that is, people not part of
+the project that still wish to contribute) to receive human consideration after
+a maximum of 24 working hours -- a desirable property for open source.
+
+Following triage, the scrum master will pull up the sprint board for an informal
+board review. This does not need to go over every single issue; rather, it
+should be an opportunity for team members to point out highlights or interesting
+updates to their tasks, and request help on blockers or otherwise.
```

Comment:
I think it would be helpful to have a "show of hands" for this part of the meeting re: how many people want to talk about a highlight or blocker. The "round the horn" process we use currently has the advantage of making it immediately obvious how many people will want to say something (i.e., the total number in the meeting) and so pacing of the meeting flows from this naturally. If an unknown number of people have comments to make during this portion, we run into the danger of either 1) the meeting being cut short before everyone gets their say, or 2) the meeting running over-time. Therefore I propose a quick 5-second check before we move into this portion of how many people will bring up a discussion item, like a mini-agenda.

Reply:
Not a bad idea! I've added that in my latest revision.